vnext/local-cli/runWindows/utils/autolink.js

```diff
@@ -322,6 +322,12 @@ async function updateAutoLink(args, config, options) {
         });
       }
 
+      if (cppPackageProviders === '') {
```

Comment:
@jonthysell: Would you mind having a peek if this is okay with you?

Reply:
This looks fine. Can you also update the cli so it gets inserted as the default in new projects? Here:

https://github.com/microsoft/react-native-windows/blob/1a936c16bcf590306659a83730a2e367710a5816/vnext/local-cli/generator-windows/index.js#L164

Reply:
Good point, thanks!

---
In reply to: [447805598](https://github.com/microsoft/react-native-windows/pull/5368#discussion_r447805598) [](ancestors = 447805598)